src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +20,163 @@
+      public static final SwerveModuleConstants S_MODULE_CONSTANTS =
+          new SwerveModuleConstants(
+              kDriveMotorCANID, kSteerMotorCANID, kBackLeftChassisAngularOffset);
+    }
+
+    /* Back Right Module - Module 3 */
+    public static final class BackRightModule {
+      // The CAN ID for the drive motor
+      public static final int kDriveMotorCANID = 6;
+      // The CAN ID for the steer motor
+      public static final int kSteerMotorCANID = 5;
+      // The Angular offsets in radians for the steer encoder
+      public static final double kBackRightChassisAngularOffset = Math.PI / 2;
+      // creating the swerve module constants for the back right module
+      public static final SwerveModuleConstants S_MODULE_CONSTANTS =
+          new SwerveModuleConstants(
+              kDriveMotorCANID, kSteerMotorCANID, kBackRightChassisAngularOffset);
+    }
+
+    public static final boolean kGyroReversed = false;
+  }
+
+  public static final class ModuleConstants {
+    // The MAXSwerve module can be configured with one of three pinion gears: 12T, 13T, or 14T.
+    // This changes the drive speed of the module (a pinion gear with more teeth will result in a
+    // robot that drives faster).
+    public static final int kDrivingMotorPinionTeeth = 14;
+
+    // Invert the turning encoder, since the output shaft rotates in the opposite direction of
+    // the steering motor in the MAXSwerve Module.
+    public static final boolean kTurningEncoderInverted = true;
+
+    // Calculations required for driving motor conversion factors and feed forward
+    public static final double kDrivingMotorFreeSpeedRps = NeoMotorConstants.kFreeSpeedRpm / 60;
+    public static final double kWheelDiameterMeters = 0.0762;
+    public static final double kWheelCircumferenceMeters = kWheelDiameterMeters * Math.PI;
+    // 45 teeth on the wheel's bevel gear, 22 teeth on the first-stage spur gear, 15 teeth on the
+    // bevel pinion
+    public static final double kDrivingMotorReduction =
+        (45.0 * 22) / (kDrivingMotorPinionTeeth * 15);
+    public static final double kDriveWheelFreeSpeedRps =
+        (kDrivingMotorFreeSpeedRps * kWheelCircumferenceMeters) / kDrivingMotorReduction;
+
+    public static final double kDrivingEncoderPositionFactor =
+        (kWheelDiameterMeters * Math.PI) / kDrivingMotorReduction; // meters
+    public static final double kDrivingEncoderVelocityFactor =
+        ((kWheelDiameterMeters * Math.PI) / kDrivingMotorReduction) / 60.0; // meters per second
+
+    public static final double kTurningEncoderPositionFactor = (2 * Math.PI); // radians
+    public static final double kTurningEncoderVelocityFactor =
+        (2 * Math.PI) / 60.0; // radians per second
+
+    public static final double kTurningEncoderPositionPIDMinInput = 0; // radians
+    public static final double kTurningEncoderPositionPIDMaxInput =
+        kTurningEncoderPositionFactor; // radians
+
```

Comment:
Ok just in general all doubles should be set as x.x using a decimal.  Floats should be x.xf.  This is the pattern you should follow for all places you set a double, from declaration to method calls.